teps/0098-workflows.md

```diff
@@ -412,85 +437,416 @@ and Pipeline definitions into a higher level opinionated syntax that is aware of
 
 ## Proposal
 
-TODO
+As proposed in [#866](https://github.com/tektoncd/community/issues/883), Pipelines as Code will be adopted into the tektoncd organization.
+It will be built and released as a Tekton project, and renamed to "Workflows", since the scope
+of the Workflows project is larger than storing Pipelines in version control.
```

Comment:
if we are talking about the project name rename, I am not against it. but we are going to need to do have to do a fair amount of work from our side to get our product using the renamed version if that happen. 

Thinking a bit, I wonder if there workflow can be an umbrella of components that can be picked up by distributor/users.

There would be things like:
pac
concurrency 
scheduling
notificatons
and other components

some sort of strong integration between all of them and common objects like RepoSitory CRD, but users would pick the thing they want.